leaguecog/leaguecog.py

```diff
@@ -344,6 +347,40 @@ async def set_summoner(self, ctx: commands.Context, name: str = "", region: str
         # re-calculate time between check games loops
         await self.calculate_cooldown()
 
+    @league.command(name="toggle-polling")
+    async def toggle_polling(self, ctx: commands.Context, state: str = None):
+        """
+        This allows the user to toggle polling on/off for their account.
+        If 'state' arg isn't passed, will check the current state and set the opposite.
+        """
+        userId = await self.bot.get_or_fetch_user(ctx.author.id)
+
+        if state:
+            # set state_bool to whatever the user entered as 'state'
+            state = state.lower().strip()
+            if state in ("on", "true"):
+                state_bool = True
+            elif state in ("off", "false"):
+                state_bool = False
+        else:
+            # get the current state of poll_user_games, and set the opposite
+            current_state = await self.config.user(userId).poll_user_games()
```

Comment:
This is unneeded I think

Reply:
I was thinking of it like a toggle switch. This way, the user doesn't have to specify the state, but they can flip it on/off based on the current state.

Reply:
It did just occur to me it needs some error handling based on user input though